pkg/version/version.go

```diff
@@ -49,3 +51,36 @@ func Get() Info {
 		Platform:   fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+// Error is an error for versions in case it is desired to check an error for this type
+type Error struct {
+	Component       string
+	ExpectedVersion string
+	Version         string
+}
+
+// Error is a required function for type Error
+func (e Error) Error() string {
+	return fmt.Sprintf("expected version: %s, found vresion: %s", e.ExpectedVersion, e.Version)
+}
+
+//Valid returns nil if expected version is meet by actual using solely Major.Minor
+func Valid(component string, actual *semver.Version, expected *semver.Version) error {
+	if actual.Major() < expected.Major() || actual.Minor() < expected.Minor() {
```

Comment:
should this be rather compare? And return -1, 0, 1?

Reply:
previously answered